bin/steps/setuptools

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+
+# Syntax sugar.
+source $BIN_DIR/utils
+
+pip-clean requirements.txt
```

Comment:
Should we have a message that tells people we're replacing stuff in their `requirements.txt` with things in the Python buildpack or overwriting their `requirements.txt` file at all.

Also, how does this affect pipenv?